app/views/users/change_email.html.erb

```diff
@@ -13,6 +13,7 @@
   <% if @user.recently_reset? %>
     <p><%= ts("You used a temporary password to log in. Please change it now as it will expire in a week.") %></p>
   <% end %>
+  <p>Your current email address is: <%= @user.email %></p>
```

Comment:
Sorry the issue wasn't entirely clear -- we meant at the top of the `<dl>`, since it fits perfectly with that structure:

```
  <dl>

    <dt><%= ts("Current Email") %></dt>
    <dd><%= @user.email %></dd>
```